corehq/apps/data_cleaning/models.py

```diff
@@ -377,15 +377,21 @@ class BulkEditChange(models.Model):
     class Meta:
         ordering = ["created_on"]
 
-    def edited_value(self, case):
+    def edited_value(self, case, edited_properties=None):
```

Comment:
Do you think it's accurate to say this shouldn't be called outside the class, and would it make sense to prefix it with an underscore?

On the other hand, it'll still be called in tests.